src/main/kotlin/com/openlattice/indexing/IndexingService.kt

```diff
@@ -90,9 +89,9 @@ class IndexingService(
 
     @Suppress("UNUSED")
     private val indexingWorker = executor.submit {
-        Stream.generate { indexingQueue.take() }
-                .parallel().forEach { entitySetId ->
-                    try {
+        while ( true ) {
+            try {
+                val entitySetId = indexingQueue.take()
                         val entitySet = entitySets.getValue(entitySetId)
```

Comment:
the pr diff is pretty but the formatting has now become ugly 😢

Reply:
see https://github.com/openlattice/indexer/pull/116#discussion_r377934256